snuba/datasets/errors.py

```diff
@@ -0,0 +1,302 @@
+                # ("ip_address", Nullable(String())),
+                # optional geo
+                # ("geo_country_code", Nullable(String())),
+                # ("geo_region", Nullable(String())),
+                # ("geo_city", Nullable(String())),
+                ("sdk_name", LowCardinality(String())),
+                ("sdk_version", LowCardinality(String())),
+                ("type", LowCardinality(String())),
+                ("version", LowCardinality(String())),
+                (
+                    "transaction_hash",
+                    Materialized(UInt(64), "cityHash64(transaction_name)"),
+                ),
+                ("span_id", Nullable(UInt(64))),
+                ("trace_id", Nullable(UUID())),
+            ]
+            + metadata_columns
+            + promoted_tag_columns
+            + [
+                # other tags
+                ("tags", Nested([("key", String()), ("value", String())])),
+                ("_tags_flattened", String()),
+                # other context
+                ("contexts", Nested([("key", String()), ("value", String())])),
+                ("_contexts_flattened", String()),
+                # http interface
+                # ("http_method", Nullable(String())),
+                # ("http_referer", Nullable(String())),
+                # exception interface
+                (
+                    "exception_stacks",
+                    Nested(
+                        [
+                            ("type", String()),
+                            ("value", String()),
+                            ("mechanism_type", Nullable(String())),
+                            ("mechanism_handled", UInt(8)),
+                        ]
+                    ),
+                ),
+                (
+                    "exception_frames",
+                    Nested(
+                        [
+                            ("abs_path", String()),
+                            ("colno", UInt(32)),
+                            ("filename", String()),
+                            ("function", String()),
+                            ("lineno", UInt(32)),
+                            ("in_app", UInt(8)),
+                            ("package", Nullable(String())),
+                            ("module", String()),
+                            ("stack_level", UInt(16)),
+                        ]
+                    ),
+                ),
+                ("culprit", String()),
+                ("sdk_integrations", Array(String())),
+                ("modules", Nested([("name", String()), ("version", String())])),
+            ]
+        )
+
+        schema = ReplacingMergeTreeSchema(
+            columns=all_columns,
+            local_table_name="errors_local",
+            dist_table_name="errors_dist",
+            mandatory_conditions=[("deleted", "=", 0)],
+            prewhere_candidates=[
+                "event_id",
+                "group_id",
+                "tags[sentry:release]",
```

Comment:
Should this be `release`?

Reply:
Not yet. The query comes with sentry:release still. In a followup I will unify them in the query processor.